observer/src/observer/constants.py

```
@@ -51,3 +51,13 @@
 
 # MCP server
 MCP_SERVER_NAME = "observer"
+
+# Container (local dev database)
+DB_CONTAINER_NAME = "observer-pg"
+DB_IMAGE = "docker.io/pgvector/pgvector:pg17"
+DB_VOLUME_NAME = "observer_data"
+DB_PORT = 5432
+DB_USER = "observer"
+DB_PASSWORD = "observer"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "DB_PASSWORD"

(S105)